tools/run-tests.py

```diff
@@ -44,7 +44,23 @@ def get_binary_path(bin_dir_path):
              '--snapshot-exec=on', '--vm-exec-stop=on', '--profile=es2015-subset']),
     Options('doctests-debug',
             ['--doctests', '--jerry-cmdline=off', '--debug', '--error-messages=on',
-             '--snapshot-save=on', '--snapshot-exec=on', '--vm-exec-stop=on', '--profile=es2015-subset'])
+             '--snapshot-save=on', '--snapshot-exec=on', '--vm-exec-stop=on', '--profile=es2015-subset']),
+    Options('unittests',
+            ['--unittests', '--jerry-cmdline=off', '--error-messages=on', '--snapshot-save=on', '--snapshot-exec=on', '--vm-exec-stop=on', '--profile=es2015-subset', '--mem-stats=on']),
```

Comment:
It seems that these lines are too long (some of them are above 120 characters). We should extract the common options to a different variable and just concatenate the rest where needed.

Reply:
Thanks for the review! Maybe we should do this to other parts of the code as well, it would make the lines much shorter and easier to read. For now I think we should just break the lines that are too long, and apply your suggestion in a separate PR afterwards.

Reply:
okay, that works for me